setup.py

```diff
@@ -13,19 +13,14 @@
       description='Mantid Total Scattering Reduction',
       long_description_content_type="text/markdown",
       license='GPL License (version 3)',
-      scripts=['bin/mantidtotalscattering'],
+      entry_points = {
+        'console_scripts': [
```

Comment:
This magic should get into addie as well.

Reply:
:+1: Working on it now. Have to move and re-structure the `main` function call (hence how I found out ADDIE is an `Image Changer` :smile:)